### PR TITLE
fix(ml,#8052): 2.7 SVM RBF accuracy ~0.95+ contradicted deterministic output 0.897

### PR DESCRIPTION
**Pull request by jsboige**
## fix(ml,#8052): 2.7 SVM RBF accuracy claim contradicted deterministic output 0.897

**Grain**: MED/code-correctness — lane myia-po-2025:CoursIA — prev: MED/code-correctness (Sudoku-5 #8244). Pivot de famille (Sudoku → ML).

### Le défaut (auto-incriminant, confirmé firsthand + determinism-gate 3×)

`2.7-Modeles-Non-Parametriques.ipynb` cellule 9 (interprétation du kernel trick) claim :

> table `| Accuracy sur make_moons | ~0.80-0.85 | ~0.95+ |`
> prose « le noyau RBF trace une frontière courbe… (accuracy **nettement supérieure**) »

Mais la **sortie réellement commitée** de la cellule de code 8 est :

```
SVM lineaire -- accuracy : 0.850
SVM RBF      -- accuracy : 0.897     <-- PAS ~0.95+
Nombre de vecteurs supports (RBF) : 100 sur 300 points d'entrainement
```

Le RBF obtient **0.897**, pas ~0.95+ — exagération de ~5 points. Et « nettement supérieure » surestime l'écart (0.897 vs 0.850 = ~4.7 pt, réel mais pas « marqué »). C'est la classe-1 #8052 : un claim d'interprétation contredit par les sorties commitées.

### Determinism-gate 3× (la sortie EST déterministe)

Tout est seedé — la sortie n'est pas un artefact de tirage aléatoire :
- cell 2 : `np.random.seed(42)`
- cell 4 : `make_moons(n_samples=300, noise=0.30, random_state=42)`
- cell 8 : `SVC(kernel="rbf", C=1.0, gamma="scale", random_state=42)`, accuracy calculée sur le **training set** (pas de train/test split)

Re-run 3× (script standalone répliquant cells 2/4/8, sklearn 1.8.0 / numpy 2.4.4) :

```
run 1: acc_lin=0.850  acc_rbf=0.897  nsv=100
run 2: acc_lin=0.850  acc_rbf=0.897  nsv=100
run 3: acc_lin=0.850  acc_rbf=0.897  nsv=100
```

Stable → le table `~0.95+` est factuellement faux, pas un drift d'exécution.

### Le fix (markdown-only, les sorties = source de vérité)

- **Table** : RBF `~0.95+` → `~0.90` (valeur linéaire `~0.80-0.85` correcte, **non touchée**).
- **Prose** : `(accuracy nettement supérieure)` → `(accuracy supérieure : 0.897 contre 0.850 pour le noyau linéaire)` — ancre l'écart mesuré, retire « nettement ».
- **Sorties inchangées** : aucune cellule de code modifiée, aucun output hand-édité (Stop & Repair — on corrige l'interprétation pour qu'elle colle aux sorties, jamais l'inverse). Pas de re-exec (le code n'a pas bougé, les sorties déterministes sont valides).

### Validation

- **Determinism-gate** 3× : acc_lin=0.850, acc_rbf=0.897, nsv=100 stables.
- **Validateur H.3** `validate_pr_notebooks.py` : PASS (10 code cells).
- **C.1/C.3 audit** `audit_c1_c3.py --family ML` : 0/0 (47 notebooks ML, 0 violation).
- **Solution-leak detector** : 0 leak sur ce notebook.
- **Diff chirurgical** : 1 fichier, +2/−2, 0 cellule de code touchée, 0 output touché, 0 CRLF, `json.dumps(indent=1)` round-trip stable.

See #8052 (audit échantillonnage — claim-vs-output mismatch, classe-1). Note : seule la cellule 9 est dérivée dans ce notebook ; les autres cellules d'interprétation (k-NN, standardisation, cancer) ne portent pas de claim numérique contredit par les sorties (vérifié firsthand).

### CODE REVIEW

**Review by clusterManager-Myia — COMMENTED** [review]
**[NanoClaw]** structural review — `fix(ml,#8052): 2.7 SVM RBF accuracy ~0.95+ contradicted deterministic output 0.897` (head `ecf4468f`, +2/-2, 1 fichier `ML/DataScienceWithAgents/02-ML-Cours/2.7-Modeles-Non-Parametriques.ipynb`, cellule markdown interprétation).

**LGTM deep** — audit-fix #8052 (claim↔output mismatch), **les deux corrections PROUVÉES firsthand contre la sortie committée** (pas juste interne-cohérentes).

**Diff = markdown-only 1 cellule** (cellule `### Interprétation : le kernel trick rendu visible`, 2 lignes swap, 0 code/output touché, code-cell count 10=10 base↔head, execution_count 1-10 byte-identical, Stop & Repair rule 6 honorée, nbformat 4.5).

**Source de vérité firsthand** : code cell idx 3 (ec=4 réel) émet la sortie littérale :
```
SVM lineaire -- accuracy : 0.850
SVM RBF      -- accuracy : 0.897
Nombre de vecteurs supports (RBF) : 100 sur 300 points
```

**Table performance corrigée** : `| Accuracy sur make_moons | ~0.80-0.85 | ~0.95+ |` → `| ... | ~0.80-0.85 | ~0.90 |`. **Vérifié firsthand** : RBF réel = **0.897** → nouveau `~0.90` est l'arrondi correct (0.897→0.90), l'ancien `~0.95+` était **faux** (surévaluation de ~5pp). Noyau linéaire réel = 0.850 → `~0.80-0.85` inchangé et correct (0.850 borne haute du range).

**Prose qualitative corrigée** : ancienne « accuracy nettement supérieure » → nouvelle « accuracy supérieure : 0.897 contre 0.850 pour le noyau linéaire ». **Vérifié firsthand** : les 2 valeurs (0.897 RBF, 0.850 linéaire) sont **byte-exact** la sortie cell 3 → la nouvelle prose est grounded, non vague. De plus le retrait de « nettement » est **pédagogiquement honnête** : le gap réel est `0.897-0.850 = 0.047` (~5pp), pas « nettement » supérieur — l'ancien qualificatif surestimait l'écart, le nouveau matche la réalité mesurée.

**Cohérence interne sound** : 100 vecteurs supports / 300 points d'entraînement = ⅓ des points deviennent SVs, cohérent avec un RBF sur demi-lunes non-linéairement séparables (frontière courbe coûteuse) — corroboré par la prose « vecteurs supports entourés en vert suffisent à la définir » (inchangée).

**0 fake output** (10/10 code cells ec 1-10 réels, 0 N/A, 0 chiffre rond suspect), **0 leak** (sk-/ghp_/AKIA/Users = 0 sur la région diff, prose math FR pure), **0 CJK** (HARD RULE Emerjesse respectée, node-regex car `grep -P` unreliable).

△ Nit mineur non-bloquant : le `~0.90` de la table est un arrondi à 1 décimale de 0.897 tandis que la prose donne `0.897` à 3 décimales — deux précisions différentes pour la même métrique, cosmétique (la prose reste la source exacte, la table la synthèse).

Audit-fix de qualité, deux corrections factuelles first-party vérifiées (la table `~0.95+` enseignait l'inverse de la réalité mesurée 0.897). — **[NanoClaw]** (myia-ai-01)
